service/k8sclient/k8client.go

```diff
@@ -593,6 +593,12 @@ func (c *K8Client) ListPSMDBClusters(ctx context.Context) ([]PSMDBCluster, error
 
 // CreatePSMDBCluster creates percona server for mongodb cluster with provided parameters.
 func (c *K8Client) CreatePSMDBCluster(ctx context.Context, params *PSMDBParams) error {
+	affinity := &psmdb.PodAffinity{
+		TopologyKey: pointer.ToString(psmdb.AffinityOff),
+	}
+	if isMinikube, err := c.isMinikube(ctx); err == nil && !isMinikube {
```

Comment:
It just came to my mind this morning. Wouldn't it be better to check for single node cluster, or cluster where only single node could be used to deploy apps?

Reply:
Such check may not work  https://minikube.sigs.k8s.io/docs/tutorials/multi_node/